docker/cygnus-ngsi/Dockerfile

```diff
@@ -0,0 +1,111 @@
+ENV GIT_REV_CYGNUS develop
+
+ENV MVN_VER "3.2.5"
+ENV MVN_TGZ "apache-maven-${MVN_VER}-bin.tar.gz"
+ENV MVN_URL "http://www.eu.apache.org/dist/maven/maven-3/${MVN_VER}/binaries/${MVN_TGZ}"
+ENV MVN_HOME "/opt/apache-maven"
+
+ENV FLUME_VER "1.4.0"
+ENV FLUME_TGZ "apache-flume-${FLUME_VER}-bin.tar.gz"
+ENV FLUME_URL "http://archive.apache.org/dist/flume/${FLUME_VER}/${FLUME_TGZ}"
+ENV FLUME_HOME "/opt/apache-flume"
+
+ENV CONF_PATH "/opt/apache-flume/conf"
+ENV CONF_FILE "/opt/apache-flume/conf/agent.conf"
+ENV AGENT_NAME "cygnus-ngsi"
+ENV LOG_LEVEL "INFO"
+ENV LOG_APPENDER "console"
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.6.0
+WORKDIR /opt
+
+RUN adduser --comment "${CYGNUS_USER}" ${CYGNUS_USER}
+
+# Install dependencies (Java, Maven, Flume)
+RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
+    yum clean all && \
+    curl --remote-name --location --insecure --silent --show-error "${MVN_URL}" && \
+    tar xzvf ${MVN_TGZ} && \
+    mv apache-maven-${MVN_VER} ${MVN_HOME} && \
+    export MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=128m" && \
+    curl --remote-name --location --insecure --silent --show-error "${FLUME_URL}" && \
+    tar zxf ${FLUME_TGZ} && \
+    mv apache-flume-${FLUME_VER}-bin ${FLUME_HOME} && \
+    mkdir -p ${FLUME_HOME}/plugins.d/cygnus && \
+    mkdir -p ${FLUME_HOME}/plugins.d/cygnus/lib && \
+    mkdir -p ${FLUME_HOME}/plugins.d/cygnus/libext && \
+    chown -R cygnus:cygnus ${FLUME_HOME}
+
+# Create Cygnus log folder
+RUN mkdir ${FLUME_HOME}/log
+
+# Clone fiware-cygnus
+RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
+    cd ${CYGNUS_HOME} && \
+    git checkout ${GIT_REV_CYGNUS}
+
+# Build and install cygnus-common
+RUN cd ${CYGNUS_HOME}/cygnus-common && \
+    ${MVN_HOME}/bin/mvn clean compile exec:exec assembly:single && \
+    cp target/cygnus-common-${CYGNUS_VERSION}-jar-with-dependencies.jar ${FLUME_HOME}/plugins.d/cygnus/libext/ && \
+    ${MVN_HOME}/bin/mvn install:install-file -Dfile=${FLUME_HOME}/plugins.d/cygnus/libext/cygnus-common-${CYGNUS_VERSION}-jar-with-dependencies.jar -DgroupId=com.telefonica.iot -DartifactId=cygnus-common -Dversion=${CYGNUS_VERSION} -Dpackaging=jar -DgeneratePom=false
+
+# Build and install cygnus-ngsi
+RUN cd ${CYGNUS_HOME}/cygnus-ngsi && \
+    ${MVN_HOME}/bin/mvn clean compile exec:exec assembly:single && \
+    cp target/cygnus-ngsi-${CYGNUS_VERSION}-jar-with-dependencies.jar ${FLUME_HOME}/plugins.d/cygnus/lib/
+
+# Install Cygnus Application script
+RUN cp ${CYGNUS_HOME}/cygnus-common/target/classes/cygnus-flume-ng ${FLUME_HOME}/bin/ && \
+    chmod +x ${FLUME_HOME}/bin/cygnus-flume-ng
+
+# Instantiate some configuration files
+RUN cp ${CYGNUS_HOME}/cygnus-common/conf/log4j.properties.template ${FLUME_HOME}/conf/log4j.properties && \
+    cp ${CYGNUS_HOME}/cygnus-ngsi/conf/grouping_rules.conf.template ${FLUME_HOME}/conf/grouping_rules.conf
+
+# Cleanup to thin the final image
+RUN cd ${CYGNUS_HOME}/cygnus-common && \
```

Comment:
Fixed in Fixed in 6fb3e11. The size of the image dramatically decreased, thanks!